lib/chitter.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'peep'
+
+class Chitter
+
+  def self.create(username, content)
+    connection = PG.connect :dbname => "chitter_#{ENV['RACK_ENV']}"
+    result = connection.exec("SELECT * FROM users WHERE username = '#{username}'").first
+    insertion = connection.exec("INSERT INTO peeps (user_id, posted_time, content) VALUES(#{result['id']}, current_timestamp, '#{content}') RETURNING id, user_id, posted_time, content").first
```

Comment:
Could think about extracting database connection to run at the beginning of running the app - and extract connection.exec into a database connection class such as [this ](https://github.com/makersacademy/course/blob/master/bookmark_manager/walkthroughs/14.md)